### PR TITLE
Fixes sitetitle for wrapper objects.

### LINKED ISSUE #1691: Seitentitel bei Sitzungsansicht => Sitzungstitel anzeigen (statt committee-1)
Der Titel wird bei Sitzungen nicht korrekt gesetzt.

### PR DESCRIPTION
**Pull request by phgross**
Customize the `object_title` method on the `plone_context_state` view for WrapperObjects, to make sure that the site title contains the Wrapper title instead of the parents id. Fixes #1691.

@lukasgraf @deiferni

### CODE REVIEW

**Comment by phgross** [comment]
FYI: ein direktes überschreiben der `Title` Methode für zu diversen Problemen beim traversieren auf Wrapperobjekte, deshalb habe ich ich dazu entschieden das Problem mit dem überschreiben der ContextStateView zu lösen.

File: opengever/meeting/tests/test_meeting_view.py [hunk 1]
```diff
@@ -126,6 +126,13 @@ def test_accessing_the_meeting_directly_shows_meeting_view(self, browser):
         self.assertEquals([u'C\xf6mmunity meeting'], browser.css('h1').text)
 
     @browsing
+    def test_site_title_is_meeting_title(self, browser):
```
Comment (deiferni) [inline opengever/meeting/tests/test_meeting_view.py]: What happens to the other content that inherits from `BaseWrapper`?

Comment (phgross) [inline opengever/meeting/tests/test_meeting_view.py]: The same thing (Title - Plone Site title), I'll add some additional checks for the other Wrapper content.

Comment (phgross) [inline opengever/meeting/tests/test_meeting_view.py]: @deiferni I've added an additional test for the member and the membership object.

**Comment by deiferni** [comment]
Looks good to me :+1:

**Pull request merged**